environment/commands/status.js

```diff
@@ -79,10 +79,25 @@ class NearContracts {
       return new Status(Unknown, Error, ContractNotFound);
     }
     try {
-      const contract = new nearlib.Contract(masterAccount, contractAccount, {});
-      // TODO #257 check contract validity here
-      //console.log(contract);
-      return new Status(Unknown, Info, NotVerified);
+      const nearAccount = new nearlib.Account(near.connection, masterAccount);
+      const contract = new nearlib.Contract(nearAccount, contractAccount, {
+        changeMethods: ["boo"],
+        viewMethods: [],
```

Comment:
i think need "initialized" in viewMethods. Also i just found only eth-client contract defined this method, we need copy it to eth-prover and mintable-fungible-token contract